joomla-extensions/weblinks
# Move lang files for search plugin

#### Summary of Changes
Move language files for the search plugin into the plugin distribution file.  This makes all the plugins consistent in not putting language files into administrator/language.

#### Testing Instructions
Upgrade an existing web links installation and make sure there are no web links language files in administrator/language/en-GB.  Make sure language strings for the search plugin are still working.
